src/api/spec/controllers/webui/patchinfo_controller_spec.rb

```diff
@@ -64,8 +64,8 @@ def do_proper_post_save
         post :new_patchinfo, params: { project: other_user.home_project }
       end
 
-      it { expect(response).to redirect_to(project_show_path(other_user.home_project)) }
-      it { expect(flash[:error]).to eq 'No permission to create packages' }
+      it { expect(response).to have_http_status(:redirect) }
+      it { expect(flash[:error]).to eq 'Sorry, you are not authorized to update this Project.' }
```

Comment:
same here `to eq('Sorry, you are not authorized to update this Project.)`

Reply:
nope, eq 'xx' is ok and we use it everywhere in the code. Doing what you are suggesting "consequently" would mean this block would look like:

``` it(proc { expect(flash[:error]).to(eq('nice text here')) })``` which isn't really what we are used to

Reply:
@vpereira I can't find any other ' eq ', so you're definitely adding an inconsistency here.

Reply:
in my branch, ```spec/models/obs_factory/staging_project_spec.rb``` and you will see it,  beside it, there are more or less 80 entries, which are similar. Reading the https://github.com/rubocop-hq/ruby-style-guide, it says that for the rspec DSL the parenthesis should'nt be used... but I'm not really into the guide, I prefer a cop which with ```rubocop  -a``` solve it and then we don't have to discuss about it :)

Reply:
the rspec DSL referenced is the .to and .not_to - look up rspec examples in there docu.

I'm all for dropping parenthesis where possible, but having nested function calls is simply hard to spot.